disperser/batcher/grpc/dispatcher.go

```diff
@@ -144,7 +144,7 @@ func GetStoreChunksRequest(blobMessages []*core.BlobMessage, batchHeader *core.B
 		if err != nil {
 			return nil, 0, err
 		}
-		totalSize += blob.BlobHeader.EncodedSizeAllQuorums()
+		totalSize += int64(blob.Bundles.Size())
```

Comment:
did you want to update the logging in L122 too?

Reply:
I think it is more useful to get workload per node, it is currently printing total blob size.